projects/addon-mobile/directives/dropdown-mobile/dropdown-mobile.template.html

```diff
@@ -3,7 +3,7 @@
     waIntersectionObserver
     waIntersectionThreshold="1"
     class="t-filler"
-    (touchstart.prevent)="close()"
+    (pointerdown.stop)="close()"
```

Comment:
There is a strange behavior on IOS:
After closing the dropdown by swiping down, it doesn't open again. Seems like some kind of pointer race

Reply:
https://github.com/user-attachments/assets/ef200d2f-f3f2-49bf-a135-33c67d48399e

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/9736/commits/677f522d83eb4e089f5d5f53894f049414c1a277